docs/api-testing/apifctl-cicd-integration.md

```diff
@@ -430,6 +435,16 @@ Identifies the ID of the event you want to see. For use with the **[`event`](#ev
 -i 123456789abc1a1abcdef123
 ```
 
+---
+### `-b <build ID>`
+<p><small>| STRING |</small></p>
+
+Adding this parameter allows you to specify the build ID you want to run your tests against. For use with the the **[`run`](#run)**, **[`run-all`](#run-all)**, **[`run-tag`](#run-tag)**, and **[`exec`](#exec)** commands. For more information, see [Test Builds](/api-testing/project-dashboard/#test-builds).
```

Comment:
Typo: "For use with the the" double "the" :)

Reply:
oopsie -- great catch! I'll get this fixed.